### PR TITLE
Minor fixes for use on (Parrot) Linux

### PR DESCRIPTION
**Pull request by gsdefender**
Hi, I had some troubles using the web module on (Parrot) Linux.
Privileges problems aside (I needed to run fit as root to get promiscuous mode) there were some minor corrections to be done in a couple of files. fit worked OK afterwards.

### CODE REVIEW

File: view/web/web.py [hunk 1]
```diff
@@ -388,6 +394,7 @@ def save_page(self):
         self.tabs.currentWidget().save_resources(self.acquisition_page_folder)
 
     def __zip_and_remove(self):
+        print(self.acquisition_page_folder)
```
Comment (Lazza) [inline view/web/web.py:397]: Can you clarify the purpose of this line?

File: view/web/web.py [hunk 2]
```diff
@@ -355,7 +357,11 @@ def __show_finish_acquisition_dialog(self):
             self.__open_acquisition_directory()
 
     def __open_acquisition_directory(self):
-        os.startfile(self.acquisition_directory)
+        if platform.system() == 'Windows':
+            os.startfile(self.acquisition_directory)
+        else:
+            subprocess.call(["xdg-open", self.acquisition_directory])
+
```
[the remark this answers is not in the document]
Comment (gsdefender) [reply view/web/web.py:364]: Do you mean `distutils.util.get_platform`? I understood it was going to be deprecated.
I can add a case for MacOS.

[the remark this answers is not in the document]
Comment (fpollicelli) [reply view/web/web.py:364]: @gsdefender I think @Lazza is referring to [def get_platform():](https://github.com/fit-project/fit/blob/37e1e8a980788a5fe17fc645090a9cacebb702df/common/utility.py#L33)

[the remark this answers is not in the document]
Comment (Lazza) [reply view/web/web.py:364]: Sorry for being unclear, I meant this one:

https://github.com/fit-project/fit/blob/37e1e8a980788a5fe17fc645090a9cacebb702df/common/utility.py#L33

**Comment by gsdefender** [comment]
I've added a call to `get_platform`.
I don't know why it returns "other" rather than "lin" on Parrot, it probably should be amended somewhat.

**Comment by fpollicelli** [comment]
> I've added a call to `get_platform`. I don't know why it returns "other" rather than "lin" on Parrot, it probably should be amended somewhat.

It seems that sys.platform is returning "linux" for Parrot, which is currently missing inside get_platform():
https://github.com/fit-project/fit/blob/37e1e8a980788a5fe17fc645090a9cacebb702df/common/utility.py#L33-L40

adding `'linux' : 'lin',` should do the trick.

**Comment by fpollicelli** [comment]
@gsdefender I applied your corrections wherever needed, thank you for your contribution!

**Review by fpollicelli — APPROVED** [review]

**Review by dpalmisano26 — APPROVED** [review]

**Pull request merged**